src/number.rs

```diff
@@ -1,7 +1,48 @@
 // Exports of <CoreFoundation/CFNumber.h>
 
+use libc::c_void;
+
 use base::*;
 
+#[repr(C)]
+struct __CFBoolean;
+pub type CFBooleanRef = __CFBoolean;
```

Comment:
Should this be `*const __CFBoolean`? `CFNumber.h` defines `CFBooleanRef` like this:

```c
typedef const struct CF_BRIDGED_TYPE(NSNumber) __CFBoolean * CFBooleanRef;
```